validator/fixtures/filters.json

```diff
@@ -200,7 +200,7 @@
             "help_text": "Click 'select' to chose the ISMN networks to use in this validation",
             "parameterised": true,
             "dialog_name": "ismn_networks",
-            "default_parameter": "AMMA-CATCH,BIEBRZA-S-1,BNZ-LTER,CARBOAFRICA,COSMOS,CTP-SMTMN,DAHRA,FLUXNET-AMERIFLUX,FMI,FR-Aqui,HOBE,iRON,LAB-net,MySMNet,OZNET,PBO-H2O,REMEDHUS,RISMA,RSMN,SCAN,SMOSMANIA,TERENO,USCRN,WEGENERNET,WSMN"
+            "default_parameter": "AMMA-CATCH,BIEBRZA_S-1,BNZ-LTER,CARBOAFRICA,COSMOS,CTP_SMTMN,DAHRA,FLUXNET-AMERIFLUX,FMI,FR_Aqui,HOBE,iRON,LAB-net,MySMNet,OZNET,PBO_H2O,REMEDHUS,RISMA,RSMN,SCAN,SMOSMANIA,TERENO,USCRN,WEGENERNET,WSMN"
```

Comment:
@wpreimes This should be updated now, I'm not sure what could be the issue here

Reply:
On my local machine it seems ok:

![image](https://user-images.githubusercontent.com/77664155/113103594-c0aa3400-91ff-11eb-9be1-f55731b81e98.png)